queries.sql

```diff
@@ -97,3 +97,48 @@ SELECT species, AVG(escape_attempts)
 FROM animals 
 WHERE YEAR(date_of_birth) BETWEEN 1990 AND 2000
 GROUP BY species;
+
+
+-- What animals belong to Melody Pond?
+SELECT a.name
+FROM animals a
+JOIN owners o ON a.owner_id = o.id
+WHERE o.full_name = 'Melody Pond';
+
+-- List of all animals that are Pokemon (their type is Pokemon).
+SELECT a.name
+FROM animals a
+JOIN species s ON a.species_id = s.id
+WHERE s.name = 'Pokemon';
+
+-- List all owners and their animals, remember to include those that don't own any animal.
+SELECT o.full_name, COALESCE(a.name, 'No animals owned') AS owned_animals
+FROM owners o
+LEFT JOIN animals a ON o.id = a.owner_id;
+
+-- How many animals are there per species?
+SELECT s.name AS species, COUNT(a.id) AS num_animals
+FROM species s
+LEFT JOIN animals a ON s.id = a.species_id
+GROUP BY s.name;
+
+-- List all Digimon owned by Jennifer Orwell.
+SELECT a.name
+FROM animals a
+JOIN species s ON a.species_id = s.id
+JOIN owners o ON a.owner_id = o.id
+WHERE s.name = 'Digimon' AND o.full_name = 'Jennifer Orwell';
+
+-- List all animals owned by Dean Winchester that haven't tried to escape.
+SELECT a.name
+FROM animals a
+JOIN owners o ON a.owner_id = o.id
+WHERE o.full_name = 'Dean Winchester';
+
+-- Who owns the most animals?
+SELECT o.full_name, COUNT(a.id) AS num_animals_owned
+FROM owners o
+LEFT JOIN animals a ON o.id = a.owner_id
+GROUP BY o.full_name
+ORDER BY num_animals_owned DESC
+LIMIT 1;
```

Comment:
**OPTIONAL** - Also, due to the duplication data I mentioned above, the answer to this query is wrong, Melomdy Poland owns the most animals (3). Please remove all duplicate data and rerun your queries so you can be sure you have the right data in your database 😄